plugins.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/wawandco/oxplugins/tools/envy"
 	"github.com/wawandco/oxplugins/tools/flect"
 	"github.com/wawandco/oxplugins/tools/grift"
+	"github.com/wawandco/oxplugins/tools/model"
```

Comment:
@dmuriel could we put this package inside the buffalo one? I think that will be good.
```suggestion
	"github.com/wawandco/oxplugins/tools/buffalo/model"
```